src/classes/CMT_MetadataAPI.cls

```diff
@@ -170,68 +192,29 @@ public class CMT_MetadataAPI {
 
             Boolean isSuccess = (result != null ? result.status == Metadata.DeployStatus.Succeeded : false);
             Map<String, String> statusMap = new Map<String, String>{ this.jobId => (result != null ? result.status.name() : 'Unknown') };
-
             // If the status is not Success, collect all errors into a string that can be written as the status
             if (result != null && result.status != Metadata.DeployStatus.Succeeded) {
-                setStatusMap(result, statusMap, isSuccess);
-            }
-
-            if (source == CMT_MetadataAPI.Source.HELP_TEXT_MIGRATION) {
-                setHelpTextMigrationSettings(statusMap, isSuccess);
-            } else {
-                setCustomizableRollupsSettings(statusMap, isSuccess);
-            }
-
-            if (isSuccess) {
-                UTIL_OrgTelemetry_SVC.asyncProcessNpspSettingsTelemetry();
-            }
-        }
-
-        /**
-        * @description Method that parses the Metadata deployment result into a map, job id => status
-        */
-        private void setStatusMap(Metadata.DeployResult result, Map<String, String> statusMap, Boolean isSuccess) {
-            String errMsg = '';
-            for (Integer n=0; n < result.details.componentFailures.size(); n++) {
-                errMsg += (n > 0 ? '\n' : '') + '* ' + result.details.componentFailures[n].fullName +
-                        '> ' + result.details.componentFailures[n].problem;
+                String errMsg = '';
+                for (Integer n=0; n < result.details.componentFailures.size(); n++) {
+                    errMsg += (n > 0 ? '\n' : '') + '* ' + result.details.componentFailures[n].fullName +
+                            '> ' + result.details.componentFailures[n].problem;
+                }
+                // The custom settings field is a max of 255 characters, but need space for the serialized json overhead
+                statusMap.put(this.jobId, errMsg.left(220));
+                isSuccess = false;
             }
 
-            // The custom settings field is a max of 255 characters, but need space for the serialized json overhead
-            statusMap.put(this.jobId, errMsg.left(220));
-            isSuccess = false;
-        }
-
-        /**
-        * @description Update Customizable Rollups Settings
-        */
-        private void setCustomizableRollupsSettings(Map<String, String> statusMap, Boolean isSuccess) {
             // Write the deployment status to the custom settings object
             Customizable_Rollup_Settings__c crlpSettings = UTIL_CustomSettingsFacade.getCustomizableRollupSettings();
             crlpSettings.CMT_API_Status__c = JSON.serializePretty(statusMap).left(255);
-
+            // THIS WILL NEED TO BE REFACTORED IF THIS CMT DEPLOYMENT CODE IS USED BY SOMETHING OTHER THAN ROLLUPS
```

Comment:
Can this comment be removed? Or is it still needed?

Reply:
This comment was left by the original dev and I think leaving it might be a good idea for the next person to hop into this class. I believe we might have have incoming changes from the Abacus folks as well

Reply:
👍🏽